15 error Error: Command failed with exit code 1: git remote update
15 error fatal: unable to access 'https://github.com/zyd19960625/lerna-demo.git/': Failed to connect to github.com port 443: Timed out
15 error error: Could not fetch origin
15 error Fetching origin
15 error     at makeError (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\execa\lib\error.js:60:11)
15 error     at Function.module.exports.sync (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\execa\index.js:194:17)
15 error     at Object.execSync (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\@lerna\child-process\index.js:39:16)
15 error     at updateRemote (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\@lerna\version\lib\is-behind-upstream.js:34:16)
15 error     at isBehindUpstream (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\@lerna\version\lib\is-behind-upstream.js:16:3)
15 error     at VersionCommand.initialize (C:\Users\zyd\AppData\Roaming\npm\node_modules\lerna\node_modules\@lerna\version\index.js:153:9)
